parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -8,35 +8,6 @@ write_files:
   content: !!binary |
     {{WrapAsVariable "provisionSource"}}
 
-{{if not .IsCoreOS}}
```

Comment:
as I'm focusing on Ubuntu, for my eyes sake I removed some CoreOS conditionals but they're not functionality important to this PoC